ui/components/app/permission-page-container/permission-page-container.component.js

```diff
@@ -158,6 +165,22 @@ export default class PermissionPageContainer extends Component {
     }
   };
 
+  footerLeftAction = () => {
+    const requestedPermissions = this.getRequestedPermissions();
+    if (requestedPermissions[PermissionNames.permittedChains] === undefined) {
+      this.goBack();
+    } else {
+      this.onCancel();
+    }
+  };
+
+  footerLeftActionText = () => {
+    const requestedPermissions = this.getRequestedPermissions();
+    return requestedPermissions[PermissionNames.permittedChains]
+      ? this.context.t('cancel')
+      : this.context.t('back');
+  };
+
```

Comment:
These are very minor adjustments  - to make the UI a bit more plausible - that wallet-ux team should come and clean up